core/src/test/java/org/mskcc/cbio/portal/util/TestStructuralVariantUtil.java

```diff
@@ -10,71 +10,68 @@
+        record.setSampleId("SMP0001");
+        record.setSvStatus("SOMATIC");
         record.setSite1EntrezGeneId(654684L);
         record.setSite1HugoSymbol("NA");
         record.setSite1EnsemblTranscriptId("ENST29O3403");
-        record.setSite1Exon(2);
+        record.setSite1Region("exon");
+        record.setSite1RegionNumber(5);
         record.setSite2EntrezGeneId(-1L);
         record.setSite2HugoSymbol("ALK1");
         record.setSite2EnsemblTranscriptId("ENST2843757657");
-        record.setSite2Exon(1);
+        record.setSite2Region("exon");
+        record.setSite2RegionNumber(7);
         Assert.assertTrue(structuralVariantUtil.hasRequiredStructuralVariantFields(record));
     }
 
     @Test
-    public void testInvalidStructuralVariantRecordWithIncompleteTranscriptInfo() {
-        // confirm that record does not meet minimum requirements for import
-        // this record is same as above but missing site 2 ensembl transcript id
-        StructuralVariant record = new StructuralVariant();
-        record.setSite1EntrezGeneId(654684L);
-        record.setSite1HugoSymbol("NA");
-        record.setSite1EnsemblTranscriptId("ENST29O3403");
-        record.setSite1Exon(2);
-        record.setSite2EntrezGeneId(-1L);
-        record.setSite2HugoSymbol("ALK1");
-        record.setSite2EnsemblTranscriptId("NA");
-        record.setSite2Exon(1);
-        Assert.assertFalse(structuralVariantUtil.hasRequiredStructuralVariantFields(record));
-    }
-
-    @Test
-    public void testInvalidStructuralVariantRecordMissingGeneInfo() {
-        // confirm that record does not meet minimum requirements for import
+    public void testValidStructuralVariantRecordMissingGeneInfo() {
+        // confirm that record does meet minimum requirements for import
         // record is missing site 1 gene info (renders record invalid)
         // and missing site 2 entrez id but present site 2 hugo symbol
         StructuralVariant record = new StructuralVariant();
+        record.setSampleId("SMP0001");
+        record.setSvStatus("SOMATIC");
         record.setSite1EntrezGeneId(Long.MIN_VALUE);
         record.setSite1HugoSymbol("NA");
         record.setSite1EnsemblTranscriptId("ENST29O3403");
-        record.setSite1Exon(2);
+        record.setSite1Region("exon");
+        record.setSite1RegionNumber(5);
         record.setSite2EntrezGeneId(Long.MIN_VALUE);
         record.setSite2HugoSymbol("ALK1");
         record.setSite2EnsemblTranscriptId("ENST2843757657");
-        record.setSite2Exon(1);
-        Assert.assertFalse(structuralVariantUtil.hasRequiredStructuralVariantFields(record));
+        record.setSite2Region("exon");
+        record.setSite2RegionNumber(7);
+        Assert.assertTrue(structuralVariantUtil.hasRequiredStructuralVariantFields(record));
     }
 
     @Test
-    public void testInvalidStructuralVariantRecordMissingGeneAndTranscriptInfo() {
+    public void testValidStructuralVariantRecordMissingSite1GeneAndTranscriptInfo() {
```

Comment:
transcriptInfo doesn't matter any more ... it does not need to be part of the test condition definition (maybe drop from filename ... or drop this whole test if it has nothing different from other cases)

yes ..this seems to cover the same situation as previous test